app/models/tslr_claim.rb

```diff
@@ -24,15 +16,10 @@ class TslrClaim < ApplicationRecord
            :claim_school,
            :employment_status,
            :current_school,
+           :subjects_taught,
+           :mostly_teaching_eligible_subjects?,
            to: :eligibility
```

Comment:
Is the intention to remove these later?

Reply:
Definitely, we want to get to a generic "Claim" model that has no eligibility-specific knowledge so it can be used for student loans and maths & physics. We've left this delegation in place to keep these PRs focused on migrating the data (which will be harder to do later) and the plan is to move them later.